.changeset/five-plums-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@solid-primitives/websocket": patch
+---
+
+websocket: rewrite
```

Comment:
might be worth including new features and others that maybe got removed / work differently